core/modules/search/templates/_results_normal.inc.php

```diff
@@ -148,7 +148,7 @@
                         <?php echo (!$issue->hasEstimatedTime()) ? '-' : \thebuggenie\core\entities\Issue::getFormattedTime($issue->getEstimatedTime(true, true)); ?>
                     </td>
                     <td class="sc_spent_time<?php if (!$issue->hasSpentTime()): ?> faded_out<?php endif; ?>"<?php if (!in_array('spent_time', $visible_columns)): ?> style="display: none;"<?php endif; ?>>
-                        <?php echo (!$issue->hasSpentTime()) ? '-' : \thebuggenie\core\entities\Issue::getFormattedTime($issue->getSpentTime(true, true)); ?>
+                        <?php echo (!$issue->hasSpentTime() || !$issue->isSpentTimeVisibleByPermission()) ? '-' : \thebuggenie\core\entities\Issue::getFormattedTime($issue->getSpentTime(true, true)); ?>
```

Comment:
The entities Issue already contain a method isSpentTimeVisible(...) that evaluate visibility of field in OR with hasSpentTime(). I think this means that if a issue has spent time without having field visible by configuration, then display it. Is required by design?
Is correct to add a new method named isTimeSpentVisible() or can create a bit of confusion?